core/src/main/java/hera/core/commands/ModuleStatus.java

```diff
@@ -0,0 +1,184 @@
+package hera.core.commands;
+
+import discord4j.core.event.domain.message.MessageCreateEvent;
+import discord4j.core.object.entity.Guild;
+import discord4j.core.object.entity.Member;
+import discord4j.core.object.entity.MessageChannel;
+import hera.core.HeraUtil;
+import hera.core.messages.MessageHandler;
+import hera.core.messages.MessageSpec;
+import hera.database.entities.Command;
+import hera.database.entities.ModuleSettings;
+import hera.database.types.LocalisationKey;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+
+import static hera.store.DataStore.STORE;
+
+public class ModuleStatus {
+	private static class CommandStatus {
+		protected Command command;
+		protected boolean isEnabled;
+		protected boolean hasPermissions;
+	}
+
+	public static Mono<Void> execute(MessageCreateEvent event, Guild guild, Member member, MessageChannel channel, List<String> params) {
+		Mono<String> message;
+		String title;
+		Mono<List<CommandStatus>> commandStatusMap = makeCommandStatusMap(getCommands(), guild);
+		if (params.size() < 1) {
+			title = HeraUtil.getLocalisation(LocalisationKey.COMMAND_MODULESTATUS_TITLE_ALL, guild).getValue();
+			message = makeMessageForAllCommands(commandStatusMap, guild);
+		} else {
+			String commandName = params.get(0);
+			//TODO add localization option option so that if you write Modules igschaltä it can work / talk to erin bout it
```

Comment:
what

Reply:
this is about comment

Reply:
soo you know i thought it would be fun if you could do aliases for optional parameter keywords
so like you can set it to $sindcommands igschaltä as alias for $modulestatus on